platform/linux-generic/pktio/dpdk.c

```diff
@@ -2406,27 +2422,26 @@ const pktio_if_ops_t _odp_dpdk_pktio_ops = {
 	.output_queues_config = dpdk_output_queues_config
 };
 
-#else
-
-#include <stdint.h>
-
-#include <odp/api/hints.h>
-
-#include <odp_packet_dpdk.h>
-#include <odp_pool_internal.h>
-
-/*
- * Dummy functions for pool_create()
- */
-
-uint32_t _odp_dpdk_pool_obj_size(pool_t *pool ODP_UNUSED, uint32_t block_size)
+static odp_bool_t is_mem_src_active(void)
 {
-	return block_size;
+	return _ODP_DPDK_ZERO_COPY;
```

Comment:
`disable_pktio` should likely also be checked here.

Reply:
v8.